File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/K. Divisors/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/K

using System;
using static System.Console;

namespace Codeforces.Assuit.Contest1
{
    class ProblemK
    {
        static void Main(string[] args)
        {
            ushort n = ushort.Parse(Console.ReadLine());
            for(int i=1;i<=n;i++){
                if(n%i==0) WriteLine(i);
            }
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/K. Divisors/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/K
#include <bits/stdc++.h>
using namespace std;
 
int main()
{
    unsigned short n;
    cin >> n;
    for(int i=1;i<=n;i++){
        if(n%i==0) cout << i << endl;
    }
}